<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>139</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>543</width>
    <height>139</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>543</width>
    <height>139</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
    <item>
     <widget class="QPushButton" name="picbutton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>109</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>145</width>
        <height>109</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,100">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>价格：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_price">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>300</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>元/月</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,9">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>面积：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_size1">
          <property name="text">
           <string>主卧（2室）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_size2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>介绍：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_info">
          <property name="text">
           <string>哈西红场 合租主卧 可月付 西典  天悦国际 公考升本推荐</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>位置：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_location">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
